src/context/UserDataContext/properties/showTags.ts

```diff
@@ -1,12 +0,0 @@
-import SimpleUserDataPropertyAPI from '../simpleUserDataPropertyAPI';
```

Comment:
The build tests are failing because `showTags` is used in other places. Maybe keep this property and remove `hideTags`? Tags should be hidden by default unless explicitly enabled.